.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - "1.10.x"
```

Comment:
Why remove the 1.10.x?

Reply:
Go standards suggest that it's users responsibility to make sure that his goversion is up to date. Leaving 1.10 in there would only increase our build time. Moreover plan is to introduce `go mod` soon, which isn't backward compatible with 1.10

Reply:
:+1